ui-core/src/types/surveys.ts

```diff
@@ -0,0 +1,65 @@
+export type VersionedForm = {
```

Comment:
"forms.ts" might be a better name for this file since it has both surveys and consents